silo-oracles/deploy/chainlink-v3-oracle/configs/sonic.json

```
@@ -357,6 +357,17 @@
         "normalizationMultiplier": 1e4,
         "invertSecondPrice": false
     },
+    "EORACLE_xUSD_dUSD": {
```

```suggestion
    "EORACLE_HARDCODED_xUSD_dUSD": {
```

we using `HARDCODED` when quote from aggregator is different from `quoteToken` from config.